lms/templates/problem.html

```diff
@@ -51,13 +51,15 @@ <h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describ
     </span>
     % endif
     </div>
-    <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
-        <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer")}</span>
-    </button>
-    <div class="submission_feedback" id="submission_feedback_${short_id}">
-    % if attempts_allowed:
-        ${_("You have used {num_used} of {num_total} attempts").format(num_used=attempts_used, num_total=attempts_allowed)}
-    % endif
+    <div class="submit-attempt-container">
+      <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
+          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer")}</span>
+      </button>
+      <div class="submission_feedback" id="submission_feedback_${short_id}">
```

Comment:
Nit: Classes should use dashes, not underscores. Looks like it was previously this way but it would be good to fix the style issue while we're in here.